idc_index/index.py

```diff
@@ -5,43 +5,33 @@
 import re
 import shutil
 import subprocess
+import sys
 import tempfile
-import urllib.request
-from pathlib import Path
+
+if sys.version_info < (3, 10):
+    from importlib_metadata import distribution
+else:
+    from importlib.metadata import distribution
 
 import duckdb
 import pandas as pd
 import psutil
 
-from ._version import version_tuple
+from . import idc_index_data
```

Comment:
After we have a standalone `idc-index-data` package versioned on its own and available through PyPI, this will be changed to `import idc_index_data`